Core/AppRuntime/Include/Babylon/AppRuntime.h

```diff
@@ -46,5 +46,8 @@ namespace Babylon
 
         std::unique_ptr<WorkQueue> m_workQueue{};
         std::function<void(const std::exception&)> m_unhandledExceptionHandler{};
+        static std::string GetStringPropertyFromError(Napi::Error error, const char* propertyName);
+        static int32_t GetNumberPropertyFromError(Napi::Error error, const char* propertyName);
+        static std::string GetErrorInfos(Napi::Error error);
```

Comment:
These should probably be anonymous local functions in the cpp instead of the header (assuming you add a new shared function for the try/catch). We just need one function that is callable by the `DefaultUnhandledExceptionHandler` implementation.

Reply:
fixed